docs/DafnyRef/Specifications.md

```diff
@@ -394,7 +394,27 @@ If a `reads` clause uses `*`, then the `reads` clause is not allowed to
 mention anything else (since anything else would be irrelevant, anyhow).
 
 A `reads` clause specifies the set of memory locations that a function,
-lambda, or iterator may read. If more than one `reads` clause is given
+lambda, or iterator may read. The memory locations are all the fields
+of all of the references given in the set specified in the frame expression
+and the single fields given in `FrameField` elements of the frame expression.
```

Comment:
It would be great to put a link to what is a framefield here. I never heard of it before.

Reply:
Done.